checker/src/org/checkerframework/checker/units/UnitsAnnotatedTypeFactory.java

```diff
@@ -217,22 +245,27 @@ private void addUnitToExternalQualMap(final Class<? extends Annotation> annoClas
                 }
 
                 // then add the aliased annotation to the alias map
-                // TODO: refactor so we can directly add to alias map, skipping the assert check in aliasedAnnotation
+                // TODO: refactor so we can directly add to alias map, skipping
+                // the assert check in aliasedAnnotation
                 aliasedAnnotation(mirror);
```

Comment:
This calls a method and ignores the returned value. aliasedAnnotation doesn't contain an assert, so this seems rather pointless. Why not call isAliasedAnnotation?